portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,60 @@
 <!DOCTYPE html>
 <html>
-  <head>
+  <html>
+<head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
-  </head>
+    </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <nav>
+        <ul>
+            <li><a href="index.html">My Portfolio</a></li>
+            <li class="active"><a href="index.html">About</a></li>
+            <li><a href="projects.html">Projects</a></li>
+            <li><a href="experiences.html">Work Experiences</a></li>
+            <li><a href="gallery.html">Gallery</a></li>
+            <li><a href="blog.html">Blog</a></li>
+        </ul>
+    </nav>
+    <div id="container">
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col">
+                    <h1>Welcome to my Portfolio!</h1>
+                    <div class="container">
+                        <a href="images/profile.jpg"><img class="image" src="images/profile.jpg"/></a>
+                    </div>
+                </div>
+            </div>
+            <br>
```

Comment:
Don't really need `<br>` for spacing, since you have a stylesheet already. You can add extra spacing like this:

```
div.row {
  margin-bottom: 2em;
}